projet/projects/views.py

```diff
@@ -106,34 +113,26 @@ def get_project_pending_contributors(request, project_id, statum):
 
 
 class CreateFeature(APIView):
-    permission_classes = [IsAuthenticated]
+    queryset = Feature.objects.all()
+    permission_classes = [
+        IsAuthenticated,
+    ]
 
     def post(self, request, project_id, format=None):
         project = Project.objects.get(id=project_id)
 
-        if (
-            Contributor.objects.filter(
```

Comment:
Moved this validation to a different class in a file so that it becomes reusable instead of repeating it in every view.